ts/features/itwallet/credentials/saga/checkCredentialsStatusAttestation.ts

```diff
@@ -80,11 +80,11 @@ export function* checkCredentialsStatusAttestation() {
     )
   );
 
-  yield* put(itwCredentialsMultipleUpdate(updatedCredentials));
+  yield* put(itwCredentialsStore(updatedCredentials));
   yield* put(
     walletAddCards(
       updatedCredentials.map(c => ({
-        key: c.keyTag,
+        key: `ITW_${c.credentialType}`,
```

Comment:
This requires a migration of the old stored credentials to the new format, otherwise we would encounter an inconsistent state of keys. Also, is the `ITW_` prefix needed? If so, should we add a comment explaining why? 

Let me know

Reply:
My bad sorry. I confused the two stores, thanks.